rest/admin_api_test.go

```diff
@@ -1470,18 +1470,23 @@ func TestReplicateErrorConditions(t *testing.T) {
 
 }
 
-//These tests validate request parameters not actual replication
-// TODO: Disabled until https://github.com/couchbase/sync_gateway/issues/3113 is fixed.  When re-enabled, 10s sleep must be removed.
-func DisableTestDocumentChangeReplicate(t *testing.T) {
+// These tests validate request parameters not actual replication.
+// For actual replication tests, see:
+// - sg-replicate mock-based unit test suite
+// - Functional tests in mobile-testkit repo
+func TestDocumentChangeReplicate(t *testing.T) {
 
 	if !base.UnitTestUrlIsWalrus() {
 		t.Skip("Skip replication tests during integration tests, since they might be leaving replications running in background")
 	}
 
 	var rt RestTester
-	defer rt.Close()
+	defer rt.Close()  // Close RestTester, which closes ServerContext, which stops all replications
 
-	time.Sleep(10 * time.Second)
+	base.EnableLogKey("Replicate")
+	base.EnableLogKey("Replicate+")
+	clog.EnableKey("Replicate")
+	clog.EnableKey("Replicate+")
```

Comment:
Can you use either base.EnableSgReplicateLogging(), base.UpdateLogKeys(...) or base.ParseLogFlags() to simplify this?

Reply:
Fixed